src/deepak/AssignmentEightWhileLoop.java

```diff
@@ -0,0 +1,62 @@
+package deepak;
+
+public class AssignmentEightWhileLoop {
+	
+	void printEvenNumbers(int start, int end) {
+		
+		System.out.println("Printing all even numbers in given range: "+ start +" TO "+ end);
+		
+		while(start <= end) {
+			if(start %2==0) {
+				System.out.println(start);
+			} start++;
+		}	
+	}
+	void printNumbersDivisibleByFive(int start, int end) {
+		
+		System.out.println("Printing all numbers divisible by five in given range: "+ start +" TO "+ end);
+		
+		while(start <= end) {
+			if(start %5 == 0) {
+				System.out.println(start);
+			} start++;
+		}	
+	}
+	void printNumbersDivisibleByFiveAndThree(int start, int end) {
+		
+		System.out.println("Printing all numbers divisible by five and three in given range: "+ start +" TO "+ end);
+		
+		while(start <= end) {
+			if(start %5 ==0 && start %3==0) {
+				System.out.println(start);
+			} start++;
+		}
+	}
+	void printNumbersDivisibleBySevenOrThirteen(int start, int end) {
+		
+		System.out.println("Printing all numbers divisible by seven and thirteen in given range: "+ start +" TO "+ end);
+		
+		while(start <= end) {
+			if(start %7 == 0 || start %13 == 0) {
```

Comment:
This can be handle in if else condition, why repetitive work we do? this looks duplicate check for same number. 
take of this, keep in mind.